src/util/handleError.ts

```diff
@@ -71,9 +71,19 @@ const getErrorLog = (
   error: ApolloError | Error | string | unknown,
   extraContext: Record<string, unknown>,
 ): ApolloError | Error | string | unknown => {
+  if (error === null) return { message: "Unknown error", ...extraContext };
```

Comment:
Sida unknown kan holde flere typer (litt på samme måte som any, men motsatt), er det mer robust å sjekke på falsy verdi heller enn å gå direkte på null?

```suggestion
  if (!error) return { message: "Unknown error", ...extraContext };
```

Reply:
> Håper vi aldri treffer denne though 😅 🤞

Enig 😅